modules/game/src/test/BinaryClockHistoryTest.scala

```diff
@@ -8,10 +8,16 @@ import chess.Centis
 
 class BinaryClockHistoryTest extends Specification {
 
-  val eps = Centis(4)
   val hour = Centis(60 * 60 * 100)
   val day = hour * 24
 
+  def beLike(comp: Vector[Centis]) = (acts: Vector[Centis]) => {
```

Comment:
This function was inspired by your help with scala tests last month, i.e.:

https://github.com/isaacl/scalachess/blob/addStats/src/test/scala/StatsTest.scala#L20-L24